### PR TITLE
Milestone issues and pull requests

### LINKED ISSUE #1711: milestone listing miss PRs
- Gitea version (or commit ref): 1.1.1
- Git version: git version 2.13.0.windows.1
- Operating system: windows
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [x] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (provide example URL) https://try.gitea.io/fiurthorn/fiurthorn/issues?state=open&milestone=51
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description

Listing open task for a milstone only lists isses, the PRs are missing.
Found no Way to list all open task for a milestone.

## Screenshots

image

image

### PR DESCRIPTION
**Pull request by lunny**
Will fix #1711

### CODE REVIEW

**Comment by lunny** [comment]
screenshot
jietu20181107-220336

**Comment by adelowo** [comment]
@lunny Would it be possible to put something like "92 opened, 10 closed" like Github ?

**Comment by lunny** [comment]
@adelowo see the screenshot, I think I have put them there and added some filters which is better than github.

**Comment by adelowo** [comment]
@lunny Sorry about that... I am so used to Github writing it in "plain text" ...

File: routers/repo/milestone.go [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+	"github.com/Unknwon/com"
+	"github.com/Unknwon/paginater"
+)
+
+const (
+	tplMilestoneIssues base.TplName = "repo/issue/milestone_issues"
+)
+
+// MilestoneIssuesAndPulls lists all the issues and pull requests of the milestone
+func MilestoneIssuesAndPulls(ctx *context.Context) {
```
[the remark this answers is not in the document]
Comment (lunny) [reply routers/repo/milestone.go]: done.

**Review by lafriks — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**